### ISSUE TITLE
collaberate - missing form labels

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#123]
WAVE is alerting to the fields below as missing form labels.
![Screen Shot 2020-10-20 at 12 06 29 PM](https://user-images.githubusercontent.com/618307/96613260-bab53c00-12cc-11eb-801a-63b58cd17fe9.png)

Here is what it says:

![Screen Shot 2020-10-20 at 12 07 29 PM](https://user-images.githubusercontent.com/618307/96613354-dae4fb00-12cc-11eb-9465-a715865bfd13.png)

**Comment by kimisgold** [comment]
I see, each of the inputs had `name` attributes, but not `id`s. That's now been fixed within the Contact Form 7 interface in the Wordpress admin.
